core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -218,6 +222,41 @@ public static void updateTableInfo(TableInfo tableInfo) {
     }
   }
 
+  public static CarbonTable buildDummyTable(
+      String tablePath,
+      String tableName,
+      String dbName,
+      boolean hasFilter) throws IOException {
+    TableInfo tableInfoInfer = CarbonUtil.buildDummyTableInfo(tablePath, tableName, dbName);
```

Comment:
what is the use of creating dummy table if alreday creating actual table down?

Reply:
removed the tableName and dbName